src/__generated__/srcQuery.graphql.js

```diff
@@ -0,0 +1,153 @@
+/**
```

Comment:
Generated code should be `.gitignore`'d

Reply:
Also `.DS_Store` should be added to `.gitignore` so that it is not put into source control.